tests/integration/test_data/private_repo_packages.yaml

```diff
@@ -0,0 +1,6 @@
+https:
```

Comment:
nitpick: It might be nice to keep the same structure as in the other test_data files and add name of the test as the first key.

Reply:
Just to clarify, do you think I should change the https one to just "private_repo"?

Reply:
no, sorry, that's not what I meant. Check out for example: https://github.com/containerbuildsystem/cachito/blob/master/tests/integration/test_data/cached_dependencies.yaml#L2
We have one key with name of the test `cached_package` and all the appropriate variables for that test are nested under that key. That way when there's another test we need to add for caching, we just use the same file for test data.

Reply:
But `private_repo_https` and `private_repo_ssh` are two different test cases, right? The structure makes sense to me

Reply:
oh right, it's fine then